lib/tasks/docker.rake

```diff
@@ -14,6 +14,8 @@
 # Other useful environment variables (not specific to this rake task)
 # => COMMIT_HASH    used by the discourse_test docker image to load a specific commit of discourse
 #                   this can also be set to a branch, e.g. "origin/tests-passed"
+# => SKIP_LINT      if using the discourse_test docker image, this will skip the docker:test task, and only run the docker:lint task
```

Comment:
no really sure what the point is of mucking with docker_test.rb ... it is so thin on logic, it was just intended a tiny script that updates the git repo and launches the rake task. 

Given this structure it allows to update the rake task and pick up the update without needing to build a new base.

I wonder if anything if the "lint/test" stuff should be removed from there. 

Conceptually the "linting" is part of testing and perhaps that should just be another flag to the test rake task.